### PR TITLE
Fix generator, add guide from JJ

### PR DESCRIPTION
**Pull request by nodar-chkuaselidze**
I should have separated these, but I noticed this problem on the way.

So fixes Generator all/file versions.

Adds guide from JJ working with Txs.

### CODE REVIEW

File: guide-generator.js [hunk 1]
```diff
@@ -40,7 +40,15 @@ if ( args.indexOf('--help') > -1 || args.length === 0 ) {
...
 }
 
-generatePages(markdownDir, all, __dirname)
+const options = {
+  all: all,
+  file: file,
+  author: author
+};
+
+//generateArchive(__dirname, markdownDir, options)
```
Comment (Bucko13) [inline guide-generator.js]: should we just delete these comments?

File: guides-markdown/working-with-txs.md [hunk 2]
```diff
@@ -0,0 +1,173 @@
+# Working with transactions
+```post-author
+Christopher Jeffrey
+```
+```post-description
+Working with transaction with bcoin, MTX and TX objects.
```
Comment (Bucko13) [inline guides-markdown/working-with-txs.md]: Change to "Working with transactions using bcoin and the MTX and TX objects."

**Comment by Bucko13** [comment]
Thanks for catching that with the `__dirname` good change with options too. Do you want to add a `post-description` for your multisig post too while we're making these changes?

File: guide-generator.js (hunk 1 above)
Comment (nodar-chkuaselidze) [reply guide-generator.js]: oh I forgot

File: guides-markdown/working-with-txs.md (hunk 2 above)
Comment (nodar-chkuaselidze) [reply guides-markdown/working-with-txs.md]: ok

File: guides-markdown/multisig-tx.md [hunk 3]
```diff
@@ -4,6 +4,10 @@
 Nodar Chkuaselidze
 ```
 
+```post-description
+Working with transactions using bcoin and the MTX and TX objects.
```
Comment (Bucko13) [inline guides-markdown/multisig-tx.md]: Post description should be something different for the multisig. Maybe: "Working with and creating multi-signature transactions using built in bcoin utilities and the wallet api."

**Pull request merged**